letsencrypt/defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+letsencrypt_webroot: "/var/www/letsencrypt"
+letsencrypt_webuser: "www-data"
+letsencrypt_webserver: "nginx"
+letsencrypt_webserver_config: "nginx/letsencrypt"
+letsencrypt_webserver_sites_available: "/etc/nginx/sites-available"
+letsencrypt_webserver_sites_enabled: "/etc/nginx/sites-enabled"
+letsencrypt_https_redirect: false
+letsencrypt_email: !!null
```

Comment:
This should default to ops@appsembler.com I think....Unless the point is that these roles are OSS...then yeah maybe not.